lib/lure/project.go

```diff
@@ -6,27 +6,32 @@ type Command struct {
 }
 
 type Project struct {
-	Vcs                string          `json:"vcs"`
-	Owner              string          `json:"owner"`
-	Name               string          `json:"name"`
-	DefaultBranch      string          `json:"defaultBranch"`
-	BranchPrefix       string          `json:"branchPrefix"`
-	TrashBranch        string          `json:"trashBranch"`
-	BasePath           string          `json:"basePath"`
-	SkipPackageManager map[string]bool `json:"skipPackageManager"`
-	Commands           []Command       `json:"commands"`
+	Vcs                 string          `json:"vcs"`
+	Owner               string          `json:"owner"`
+	Name                string          `json:"name"`
+	DefaultBranch       string          `json:"defaultBranch"`
+	BranchPrefix        string          `json:"branchPrefix"`
+	TrashBranch         string          `json:"trashBranch"`
+	BasePath            string          `json:"basePath"`
+	SkipPackageManager  map[string]bool `json:"skipPackageManager"`
+	UseDefaultReviewers *bool	    `json:"useDefaultReviewers"`
+	Commands           []Command        `json:"commands"`
 }
 
 type LureConfig struct {
 	Projects []Project `json:"projects"`
 }
 
 const (
-	defaultBranchPrefix  string = "lure-"
-	defaultTrashBranch   string = "closed-branch-trash"
-	defaultCommitMessage string = "Update {{.module}} to {{.version}}"
+	defaultBranchPrefix  string 	= "lure-"
+	defaultTrashBranch   string 	= "closed-branch-trash"
+	defaultCommitMessage string 	= "Update {{.module}} to {{.version}}"
 )
 
+func newTrue() *bool {
+    b := true
+    return &b
+}
```

Comment:
its a bit awkward but this was the internet's best advice. I had to make the struct field a pointer to allow it to be optional because I want the default to be true -_-